custom-deploy-steps/get-image-uri/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Get image URI
+description: Get image uri from AWS ECR
+
+inputs:
+  ecr_repository:
+    description: ECR repository to pull image from
+    required: true
+  aws_region:
+    description: AWS region
+    required: true
+
+outputs:
+  image-uri:
+    description: "Image URI"
+    value: ${{ steps.get-image-uri.outputs.image-uri }}
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+
+    - name: Get Image URI
+      shell: bash
+      id: get-image-uri
+      run: |
+        # For testing purposes ( actual value: =master-${{ github.sha }} )
```

Comment:
We need to remember to replace this before merge. There are also other places in the code that are currently set up for testing. I'm leaving the comment here as a checklist to update before merge:

 - [ ] replace @feature/... with @master in actions
 - [ ] replace code changes only made for testing 
 
 Please only resolve this comment once everything has been updated and we are ready to merge.